code/game/antagonist/antagonist_helpers.dm

```diff
@@ -1,6 +1,8 @@
 /datum/antagonist/proc/can_become_antag(var/datum/mind/player, var/ignore_role)
 
 	if(player.current)
+		if(isliving(player.current) && player.current.stat)
+			return 0
```

Comment:
Should probably check for the override mob flag in here, as antags which override mobs may as well pull from any kind of mob.